File: Service/Admin/RMA.Service.Admin.SecurityManager.Contracts/Entities/UserCompanyMap.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.SecurityManager.Contracts.Entities
{
    public class UserCompanyMap : AuditDetails
    {
        public int UserCompanyMapId { get; set; }
        public int? UserId { get; set; }
        public int CompanyId { get; set; }
        public int RoleId { get; set; }
        public UserCompanyMapStatusEnum UserCompanyMapStatus { get; set; }
        public int? UserActivationId { get; set; }

        public string UserName { get; set; }
        public string DisplayName { get; set; }
        public string RoleName { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/member-manager/models/user-company-map.ts
import { UserCompanyMapStatusEnum } from 'projects/shared-models-lib/src/lib/enums/user-company-map-status-enum';

export class UserCompanyMap {
    userCompanyMapId: number;
    userId: number;
    companyId: number;
    roleId: number;
    userCompanyMapStatus: UserCompanyMapStatusEnum;
    userActivationId: number;

    isDeleted: boolean;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;

    userName: string;
    displayName: string;
    roleName: string;
}
